pkg/providers/instancetype/zz_generated.bandwidth.go

```diff
@@ -20,56 +20,45 @@ package instancetype
 
```

Comment:
You might need a rebase?

Reply:
Changes in this file are the result of me running `make codegen` as directed by the [Developer Loop](https://karpenter.sh/docs/contributing/development-guide/#build-and-deploy) section of the development-guide.

Should reverse these changes?

Reply:
Yes. If this is in our Developer Loop section, we should pull it out. This doesn't need to be run on every code push now.

Reply:
Fixed in next revision.